exampleSite/hugo.toml

```diff
@@ -15,10 +15,10 @@ theme = "adritian-free-hugo-theme"
   vercelPageInsights = false
   vercelAnalytics = false
```

Comment:
Let's move the vercel analytics also to this section, as well as page insights :) 
(I'll make a PR in your original branch)

Reply:
Done in 22f7d0e3eba8d51d756646f0617935a62b5f9e6f